modules/weko-admin/weko_admin/utils.py

```diff
@@ -449,8 +451,8 @@ def write_report_tsv_rows(writer, records, file_type=None, other_info=None):
         records = list(records.values())
     for record in records:
         if file_type is None or \
-                file_type == 'file_download' or \
-                file_type == 'file_preview':
+            file_type == 'file_download' or \
```

Comment:
Consider to use:
list_of_type = [None, 'file_download', 'file_preview']
if file_type in list_of_type:
    # TODO